lcls-twincat-pmps/PMPS/HelperFunctions/F_eVRangeCalculator.TcPOU

```diff
@@ -41,7 +41,7 @@ END_VAR
 
 MaxEv := Boundaries[PMPS_GVL.g_cBoundaries];
 
-IF reV < 0 THEN F_eVRangeCalculator := 16#FFFF; RETURN; END_IF // Failsafe
+IF reV < 0 OR reV > Boundaries[PMPS_GVL.g_cBoundaries] THEN F_eVRangeCalculator := 16#FFFF; RETURN; END_IF // Failsafe
```

Comment:
Two questions:
- Why not use `MaxEv` defined two lines above?
- What affect does setting the bitmask to FFFF have? Does that tell everyone that the beam is at every energy as a failsafe?

Reply:
Fixed point 1.
Yes, using FFFF is safest as it's recognizably impossible, and also produces a residual everywhere. Operating above the max ev is unexpected.